src/inmanta/server/services/orchestrationservice.py

```diff
@@ -68,6 +68,9 @@
 
 LOGGER = logging.getLogger(__name__)
 
+PERFORM_CLEANUP = True
```

Comment:
```suggestion
PERFORM_CLEANUP: Literal[True] = True
```
If I understand correctly this should always be true except for temporary use cases?

Reply:
yes, but I'm going to make it a bool anyways

Reply:
I can live with that.